### PR TITLE
Extract common run_cmd to prevent duplication

### PR DESCRIPTION
**Pull request by okurz**
Verification run: http://lord.arch.suse.de/tests/2037

### CODE REVIEW

**Comment by okurz** [comment]
Need to remove "[WIP] Add debugging script for remote ssh command execution" again

**Comment by okurz** [comment]
> Makes sense. So what's missing form your point of view (WIP is still in title)?

Manual integration test. Let's see how http://lord.arch.suse.de/tests/2037 turns out.

> When I previously looked at this I spotted a test script which you seem to have removed. 
> Maybe you can actually share this somehow? On the first look it seemed useful.

Well, I used it only to debug but I made it available on https://gist.github.com/okurz/1e82cdecf6e5c478119a58964117e119 if you are interested.

File: consoles/sshVirtsh.pm [hunk 1]
```diff
@@ -505,17 +506,20 @@ sub add_disk {
     return;
 }
 
+sub virsh {
+    my $libvirt_connector = get_var('VMWARE_REMOTE_VMM', ' ');
+    return "virsh$libvirt_connector";
```
Comment (mnowaksuse) [inline consoles/sshVirtsh.pm]: `VMWARE_REMOTE_VMM`, if defined, may not start with space:
```suggestion
    return "virsh $libvirt_connector";
```

File: backend/svirt.pm [hunk 2]
```diff
@@ -72,13 +72,13 @@ sub do_stop_vm {
...
+            $self->run_ssh_cmd("$ps Stop-VM -Force -VMName $vmname -TurnOff");
+            $self->run_ssh_cmd("$ps Remove-VM -Force -VMName $vmname");
         }
         else {
-            my $libvirt_connector = get_var('VMWARE_REMOTE_VMM', '');
-            $self->run_cmd("virsh $libvirt_connector destroy $vmname");
-            $self->run_cmd("virsh $libvirt_connector undefine --snapshots-metadata $vmname");
+            my $libvirt_connector = get_var('VMWARE_REMOTE_VMM', ' ');
+            $self->run_ssh_cmd("virsh$libvirt_connector destroy $vmname");
+            $self->run_ssh_cmd("virsh$libvirt_connector undefine --snapshots-metadata $vmname");
```
Comment (mnowaksuse) [inline backend/svirt.pm]: Space as well?

File: consoles/sshVirtsh.pm (hunk 1 above)
Comment (okurz) [reply consoles/sshVirtsh.pm]: true, updated. I hope I got it right.

**Comment by pevik** [comment]
I don't like the naming `run_cmd` and `run_ssh_cmd`: `run_cmd` is also working with `ssh` via ssh channel.
\+ `run_cmd` uses actually only ssh channel, so it might be better to pass it instead of `ssh`

**Pull request merged**